devel/jmeter/Portfile

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       java 1.0
+
+name            jmeter
+categories      devel benchmarks java net
+platforms       darwin
+license         Apache-2.0
```

Comment:
```suggestion
license         Apache-2
```

Reply:
How to understand how to specify the license? It's a little confusing on this page: https://trac.macports.org/wiki/PortfileRecipes#licensekeyword